static/scripts/topicEdit.js

```diff
@@ -184,7 +185,7 @@ const SortableList = SortableContainer(({
         <div>
             {items.map((value, index) => (
                 <SortableItem
-                    key={`item-${index}`}
+                    key={`item-${value._id ? value._id : value.key}`}
```

Comment:
Why do we need the distinction? In which cases do we have an `_id` but not an `key` or or vice versa?

Reply:
If you add new editor that is not yet saved then there is not _id  but I am assigning id with the help of shortid library. Then when you saved it and reload e.g. the value._id is present since it comes from the backend

Reply:
Thank you for the clarification